src/main/resources/avro/readmethods.avdl

```diff
@@ -12,6 +12,11 @@ This request maps to the body of `POST /reads/search` as JSON.
 If a reference is specified, all queried `ReadGroup`s must be aligned
 to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
+
+_Error Conditions_
+
+- If `readGroupIds` is empty, this method will throw `GAException` and set the HTTP
```

Comment:
Could we start a page in the documentation branch that generally describes
what HTTP errors should be uses in what conditions?  

Jerome Kelleher <notifications@github.com> writes:
> In src/main/resources/avro/readmethods.avdl:
> 
> > @@ -12,6 +12,11 @@ This request maps to the body of `POST /reads/search` as JSON.
> >  If a reference is specified, all queried `ReadGroup`s must be aligned
> >  to `ReferenceSet`s containing that same `Reference`. If no reference is
> >  specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
> > +
> > +_Error Conditions_
> > +
> > +- If `readGroupIds` is empty, this method will throw `GAException` and set the HTTP
> 
> I don't think this is a 501 condition. This is Bad Request, so should be a 400.
> Or maybe something more specific? I can't see any code that fits this condition
> better than 400 at http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html
> 
> —
> Reply to this email directly or view it on GitHub.*
>